demo-shell-ng2/app/components/activiti/activiti-demo.component.ts

```diff
@@ -190,13 +191,16 @@ export class ActivitiDemoComponent implements AfterViewInit {
     }
 
     navigateStartProcess() {
+        this.processFilter = null;
```

Comment:
I see this 2 lines be repeated twice, can we put it in a private method that explains what they do?